flight/mcu/lib/teensyShared/dataLog.cpp

```diff
@@ -145,47 +152,67 @@ void logData(File* dataFile, sensorDataStruct_t* data) {
 }
 
 bool DataLogBuffer::pushLowGFifo(LowGData* lowG_Data) {
```

Comment:
Every function must have a doxygen comment. I know I'm asking for a lot of commenting changes, but hey if all of avionics is doing a documentation push I guess this is the software documentation push.

Reply:
I feel like the push[something]Fifo functions are really obvious in terms of what they do. They're single-line functions and adding doxygen comments for each one doesn't feel like good practice. I agree that all non-obvious functions should have doxygen comments though